SampleApplication/Startup.cs

```diff
@@ -31,6 +31,18 @@ public void ConfigureServices(IServiceCollection services)
                         new JsonMessageDeserializer<Message>()),
                 types => types.FromAssemblyOf<Startup>());
 
+            // services.AddKafkaListener(
```

Comment:
What is Listener? Too much lambdas inside

Reply:
wil discuss it in another PR